cmd/reva/import.go

```diff
@@ -40,133 +64,115 @@ func importCommand() *command {
 			fmt.Println(cmd.Usage())
 			os.Exit(1)
 		}
-		f := cmd.Args()[0]
+		exportPath := cmd.Args()[0]
 
-		sClient, err := getStorageProviderClient()
-		if err != nil {
-			return err
-		}
-		uClient, err := getUserShareProviderClient()
-		if err != nil {
-			return err
-		}
+		return importShares(exportPath)
+	}
+	return cmd
+}
 
-		ctx := getAuthContext()
+//Imports shares from an shares.jsonl file in exportPath. The files must already be present on the storage
+func importShares(exportPath string) error {
+	storage, err := getStorageProviderClient()
+	if err != nil {
+		return err
+	}
+	sharing, err := getUserShareProviderClient()
+	if err != nil {
+		return err
+	}
+	ctx := getAuthContext()
+	sharesJSONL, err := os.Open(path.Join(exportPath, "shares.jsonl"))
+	if err != nil {
+		log.Fatal(err)
```

Comment:
following the previous comment, return the error here and deal with it in the caller